depthai_ros_driver/include/depthai_ros_driver/depthai_common.hpp

```diff
@@ -28,12 +28,8 @@
 #include <depthai/nnet/nnet_packet.hpp>
 #include <depthai/pipeline/cnn_host_pipeline.hpp>
 
-// std includes
-#include <algorithm>
-#include <array>
-#include <memory>
-#include <string>
-#include <variant>
+#include <depthai_ros_driver/ros_agnostic.hpp>
+#include <depthai_ros_driver/impl/ros_agnostic.hpp>
```

Comment:
Thoughts on including the impl in the `<depthai_ros_driver/ros_agnostic.hpp>` so this line is not needed in downstream usage?

Reply:
I noticed the impl was included. Shouldn't the main include file be <depthai_ros_driver/ros_agnostic.hpp>? cc: @gbiggs

Reply:
Addressed https://github.com/osrf/depthai_ros/pull/4/commits/c04922d9f1b691c8fd58c29c0b0c7abc286c45a4